CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Change Log
+
+v0.2.0 **Updated README.md to new format** (2020-05-29)
+
+- Also externalised this change log to CHANGELOG.md
+
+v0.1.1 **Updated eslint dependency version** (2020-05-29)
+
```

Comment:
Should we always include at least one descriptive line below the version/title or should we omit it if it will be identical to the title?

Reply:
I struggled with that too and decided that omitting is best. Some changes, e.g. majors and minors should get a decent description. A patch is typically very small and can likely be summarised in just the title/sentence (referencing my earlier comment questioning whether we need titles or just go with a descriptive sentence).

Reply:
Sounds good, omitting it is.